scipy/optimize/__init__.py

```diff
@@ -86,6 +86,7 @@
 
    leastsq - Minimize the sum of squares of M equations in N unknowns
    nnls - Linear least-squares problem with non-negativity constraint
+   lsq_linear - Linear least-squares problem with arbitrary bound constraints
```

Comment:
Yes, once you're in context, it's fine. In the general listing of routines though, "arbitrary bounds" sounds too general. It's too easy to mistake this for arbitrary shaped regions of n-dimensional space.